js/trek.js

```diff
@@ -0,0 +1,120 @@
+// Wave 3: While viewing a single trip, you can reserve a spot
+// Use a form to submit your name to reserve a spot on the trip you are viewing.
+
+$(document).ready(() => {
+
+  $('.tripView').hide();
+  $('.reveal').hide();
```

Comment:
Instead of putting this in `$(document).ready`, a better strategy is to mark them as `display: none;` in your CSS (this is what `.hide()` does behind the scenes). That way they won't flash on the screen for a moment before disappearing.